lib/all_unique.js

```diff
@@ -1,8 +1,18 @@
 module.exports = {
   all_unique: function(str){
-  
+  var separatedString = str.slice(0);
```

Comment:
It's very hard to look for where that error might be coming from without proper indentation. Please indent and if you are still having issues, commit and push again.